tests/server/db/foods.test.js

```diff
@@ -24,6 +24,21 @@ test('db.getFoods returns an array of 27 foods', () => {
   //   })
 })
 
+test('db.addFood should add food to db', () => {
```

Comment:
Can this be underneath `getFood` too please?
Just for consistency and so our simpler fns are at the top. Cheers 👍